### PR TITLE
add php/elixir

### PR DESCRIPTION
**Pull request by neenjaw**

### CODE REVIEW

File: test/languages/php_test.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    class CsharpTest < Minitest::Test
+      def test_full_example
+        code = <<~CODE
+          <?php
+
+          namespace Lib\\Exercism\\Test;
+
+          use Lib\\NotExercism;
+
+          // I love C#!!
```
Comment (neenjaw) [inline test/languages/php_test.rb:14]:
```suggestion
          // I love C#!! But get paid with PHP
```

File: test/languages/elixir_test.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+
+module SnippetExtractor
+  module Languages
+    class RubyTest < Minitest::Test
```
Comment (iHiD) [inline test/languages/elixir_test.rb:5]:
```suggestion
    class ElixirTest < Minitest::Test
```

File: test/languages/elixir_test.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          end
+        CODE
+
+        expected = <<~CODE
+          defmodule TwoFer do
+            ...
+          end
+        CODE
+
+        assert_equal expected, ExtractSnippet.(code, :ruby)
```
Comment (iHiD) [inline test/languages/elixir_test.rb:27]:
```suggestion
        assert_equal expected, ExtractSnippet.(code, :elixir)
```

File: test/languages/php_test.rb [hunk 4]
```diff
@@ -0,0 +1,63 @@
+require "test_helper"
+
+module SnippetExtractor
+  module Languages
+    class CsharpTest < Minitest::Test
```
Comment (iHiD) [inline test/languages/php_test.rb:5]:
```suggestion
    class PhpTest < Minitest::Test
```

File: test/languages/php_test.rb [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+              function total()
+              {
+                  return array_reduce(range(1, 64), function ($acc, $n) {
+                      return $acc += square($n);
+                  });
+              }
+          }
+        CODE
+
+        assert_equal expected, ExtractSnippet.(code, :csharp)
```
Comment (iHiD) [inline test/languages/php_test.rb:59]:
```suggestion
        assert_equal expected, ExtractSnippet.(code, :php)
```

**Comment by iHiD** [comment]
Nailed it in one! Nice work :)

**Pull request merged**